.github/workflows/release.yml

```diff
@@ -33,16 +33,23 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-go-
 
+      - name: Login to Upbound
```

Comment:
Crossplane support said that it's enough to login via docker but in my tests I had to login via up CLI as well in order to be able to push the package.